overrides.nix

```diff
@@ -920,6 +951,22 @@ self: super:
     }
   );
 
+  pyproj = super.pyproj.overridePythonAttrs (
+    old: {
+      buildInputs = (old.buildInputs or [ ]) ++
+        [ self.cython ];
+      PROJ_DIR = "${pkgs.proj}";
+      PROJ_LIBDIR = "${pkgs.proj}/lib";
+      # pkgs.proj doesn't export the required headers, so grab them from the source repo
+      PROJ_INCDIR = "${pkgs.fetchFromGitHub {
```

Comment:
This needs to be changed to something version independent. We don't want to maintain fixed upstreams in our overrides.

Preferably you'd fix this in nixpkgs to include the relevant headers and only then use it in poetry2nix.
Feel free to CC me on any nixpkgs PR.

Reply:
Turns out `pks.proj.dev` has the headers in it already :tada: